app/views.py

```diff
@@ -48,17 +54,7 @@ def play_music():
 @app.route('/saved-videos', methods=['GET'])
 def saved_videos():
     if 'google_token' in session:
-        playlist_titles = viewsModel.getplaylisttitles(session[
-            'session_user_id'])
-        playlist_tracks = []
```

Comment:
Here's another situation where going down the unused-variable rabbit hole revealed that this whole block is, apparently, dead code. `playlist_tracks` is not used, and thus all of the code below is not doing useful work in populating it. Either we are relying on a hidden side effect in it, which is extremely dangerous, or we don't need to keep any of this logic here.